Maple2Storage/Types/Metadata/ScriptMetadata.cs

```diff
@@ -127,29 +127,26 @@ public override string ToString()
     public class Content
     {
         [XmlElement(Order = 1)]
-        public long Script;
```

Comment:
Why is this removed? `Script` must have an Id.

Reply:
You are right, it was a mistake.

Reply:
The script in Content refers to a long that the server doesn't need to know about it.
Example: 
```
    <script randomPick="1" id="10" feature="" locale="">
		<content voiceID="ko/Npc/00000079" **text="$script:0831180407000003$"** />
	</script>
```
The important part is ID for the Element Script not the Content Script.